Cargo.toml

```diff
@@ -12,7 +12,7 @@ repository = "https://github.com/webrtc-rs/turn"
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-util = { package = "webrtc-util", version = "0.1.18" }
+util = { package = "webrtc-util", version = "0.1.18", features = [ "vnet" ] }
```

Comment:
as mentioned in comments to https://github.com/webrtc-rs/util/pull/7, we don't need to change it now unless necessary.